ietf-layer0-types.yang

```diff
@@ -1550,14 +1546,75 @@ module ietf-layer0-types {
         container explicit-mode {
           config false;
           description
-            "The set of attributes for an explicit mode";
-          // uses explicit-mode;
+            "The set of attributes for an explicit mode.
+
+            The explicit mode attributes can be defined either:
+            - as a leafref to an explicit mode template; or:
+            - using the explicit-mode grouping.
+
+            Therefore, this container shall be augmented with either
+            a use statement for the explicit-mode grouping or a
+            leafref to an explicit mode template and the choice
+            depends on where this grouping is used.
+            In the latter case, the augmentation shall provide the
+            correct XPaths that depend on where this grouping is used
+            within the YANG data tree.
+            See for example the YANG code commented below";
+/*  YANG example for using the explicit-mode grouping:
```

Comment:
Is it a best practice to include commented YANG code? I was thinking that it'd better to include it inside the description statement, but looking at the next change below it is rather difficult to do.
Is there really a need to include these example YANG code in the YANG, or rather we put it in the draft text as an appendix?

Reply:
See: https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext-RFC9093-bis/pull/113/commits/f80e9d5e2eb2b64ab42880993938647b04aedbfd